src/internal_events/http.rs

```diff
@@ -60,6 +98,7 @@ impl InternalEvent for HttpEventMissingMessage {
 
     fn emit_metrics(&self) {
         counter!("events_discarded_total", 1);
```

Comment:
I'm assuming this counter is kept to avoid breaking changes. If so, when/how would we eventually go about deprecating these?

Reply:
Correct. We will leave the old counters in for a version or two, I expect, marking them as deprecated once all components have the new values, and then remove them.